can-view-callbacks.js

```diff
@@ -5,6 +5,8 @@ var getGlobal = require('can-util/js/global/global');
 var domMutate = require('can-util/dom/mutate/mutate');
 var namespace = require('can-namespace');
 
+var GLOBAL = getGlobal();
```

Comment:
@phillipskevin Is it alright to initialize this once and save it? I noticed line 72 calls it each time it's used.

Reply:
It's definitely ok to cache it within the `tag` function. I'm not sure if it will cause problems (in the stache / bindings tests) to cache it when the module is first imported.

Reply:
It's definitely ok to cache it within the tag function. I'm not sure if it will cause problems (in the stache / bindings tests) to cache it when the module is first imported.